### PR TITLE
Adjust the calculation to handle path with 8 characters

### PR DESCRIPTION
**Pull request by akrambek**

### CODE REVIEW

File: src/main/java/org/reaktivity/nukleus/http/internal/util/HttpUtil.java [hunk 1]
```diff
@@ -101,7 +101,7 @@ public static boolean isPathValid(
         if (valid)
         {
             byte_loop:
-            for (int index = (originalCapacity < 8) ? 0 : capacity; capacity > 0; index++, capacity--)
+            for (int index = (originalCapacity <= 8) ? 0 : capacity; capacity > 0; index++, capacity--)
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/http/internal/util/HttpUtil.java:104]: If we make this change, then it works, but it forces the `8 byte` ascii string to use the less efficient per-byte approach instead of the 8-byte stride in the main loop above.

If we revert this change, but instead change main loop to be `capacity >= Long.BYTES` then we get the benefit of the 8-byte striding approach for `8 byte` ascii strings, agree?

Comment (akrambek) [reply src/main/java/org/reaktivity/nukleus/http/internal/util/HttpUtil.java:104]: Yeah much better.
